### DISCUSSION TITLE
[GitHub] two-factor recovery codes

### DISCUSSION THREAD

**Discussion by geisabitt** [community/community#102041] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

The two-factor authentication is unnecessary and should not be mandatory to keep the GitHub account active.

**Comment by ghostinhershell** [comment]
Hey there! We know 2FA may be a new process for some folks and with that comes questions. We put together a [FAQ](https://github.com/orgs/community/discussions/68240) that will help answer your concerns above. If you still have more questions, feel free to ask them here.

**Reply by geisabitt** [reply]
It's useless and complicated, I already registered this, but this should be optional and not mandatory
